packages/router/src/grpc/view-protocol-server/gas-prices.ts

```diff
@@ -0,0 +1,13 @@
+import type { Impl } from '.';
+
+export const gasPrices: Impl['gasPrices'] = () => {
+  /** @todo Replace this stub with real gas prices. */
```

Comment:
@Valentine1898 is currently working on this as part of #203

Reply:
It seems that gas prices do not actually change on the network and are always equal to these values
```
blockSpacePrice: 0n
compactBlockSpacePrice: 0n
executionPrice: 0n
verificationPrice: 0n
```

Reply:
@Valentine1898 but that could change in the future, right? i.e., I should still be requesting them (using your new implementation) rather rather than depending on all-0 values?

Reply:
Yeah, I was just clarifying that we're currently getting all the 0 values from the network

Reply:
Gotcha, thanks